src/Tizen.NUI/src/public/ParticleSystem/ParticleEmitter.cs

```diff
@@ -0,0 +1,575 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.ComponentModel;
+
+namespace Tizen.NUI.ParticleSystem
+{
+    using Tizen.NUI.BaseComponents;
+
+    /// <summary>
+    /// Enum defining blending options when rendering the particles.
+    /// </summary>
+    public enum ParticleBlendingMode
```

Comment:
```suggestion
    [EditorBrowsable(EditorBrowsableState.Never)]
    public enum ParticleBlendingMode
```

We should also mark public enums as 'hidden' if we are not prepare to ACR.